runtime/oti/j9nonbuilder.h

```diff
@@ -5635,6 +5635,8 @@ typedef struct JFRState {
 	IDATA blobFileDescriptor;
 	void *jfrWriter;
 	UDATA jfrChunkCount;
+	U_64 chunkStartTime;
```

Comment:
Are the fields in the output signed or unsigned?

Reply:
The Java APIs will read them as signed values